syntax = "proto3";

package example;

message EchoRequest {
    string message = 1;
}

message EchoResponse {
    string message = 1;
}

message AddRequest {
    int32 a = 1;
    int32 b = 2;
}

message AddResponse {
    int32 result = 1;
}